server/routes/testutils/appSetup.ts

```diff
@@ -51,6 +52,7 @@ function appSetup(services: Services, production: boolean): Express {
   app.use('/monitor', monitorRoutes(services))
   app.use('/dependencies', dependencyRoutes(services))
   app.use('/product-dependencies', productDependenciesRoutes(services))
+  app.use('/github-repo-request-form', formsRoutes(services))
```

Comment:
As Andy said, this needs to match the include in app.ts so should be `/forms`

Reply:
Resolved